packages/lib/src/components/Card/components/CardInput/components/ExpirationDate.tsx

```diff
@@ -56,6 +57,19 @@ export default function ExpirationDate(props: ExpirationDateProps) {
                     }
                 )}
             />
+            <div className={classNames(
+                'adyen-checkout__field__exp-date_hint_wrapper',
+                [styles['checkout__field__exp-date_hint_wrapper']],
+                {
+                    'adyen-checkout__field__exp-date_hint_wrapper--hidden': error || isValid,
+                }
+            )}>
+                <img
+                    src="https://checkoutshopper-test.adyen.com/checkoutshopper/images/components/expiry_date_hint.svg"
+                    className="adyen-checkout__field__exp-date_hint"
+                    alt={fieldLabel}
+                />
+            </div>
         </Field>
```

Comment:
Why aren't we implementing the same way as the CVC hint image in this case?

Reply:
I believe it's to do with the fact that the cvc icon animates, so is an inline svg. Whilst the date icon does not - so we load it from the server